Code/MinimalLib/jswrapper.cpp

```diff
@@ -183,6 +183,53 @@ JSReaction *get_rxn_no_details(const std::string &input) {
   return get_rxn(input, std::string());
 }
 
+std::string generate_aligned_coords_deprecated(JSMol &self,
```

Comment:
These deprecations should probably be mentioned in the corresponding section of the release notes

Reply:
Done.